modules/ROOT/pages/mongodb/mongodb-connector-6-0-design-center.adoc

```diff
@@ -0,0 +1,78 @@
+= MongoDB Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+Design Center enables you to create apps visually. To use Design Center, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Design Center:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+For information on Design Center field values, see the xref:mongodb/mongodb-connector-reference.adoc[MongoDB Connector Reference].
```

Comment:
GLOBAL
```suggestion
For information about Design Center field values, see the xref:mongodb/mongodb-connector-reference.adoc[MongoDB Connector Reference].
```
Should we retitle this deliverable to the proper connector name: Anypoint Connector for MongoDB Reference?

Reply:
The Reference book titles do not have the full name, so I am leaving this.

Reply:
This is not titled that way in the Reference Doc.